ice/agents/openai.py

```diff
@@ -31,13 +31,14 @@ async def answer(
         prompt: str,
         multiline: bool = True,
         verbose: bool = False,
-        default: str = "",
```

Comment:
I don't think this can be removed here unless it's removed in all agents.

Reply:
ah ok, sounds plausible!